webblog/blog/models.py

```diff
@@ -0,0 +1,9 @@
+from django.db import models
+
+
+class Blog(models.Model):
+    title = models.CharField(max_length=100, blank=True, default='')
+    description = models.CharField(max_length=1500)
```

Comment:
We should use `TextField` for `description`.

Reply:
done